pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -327,8 +330,8 @@ func NewCertRotationController(
 		certrotation.TargetRotation{
 			Namespace: operatorclient.TargetNamespace,
 			Name:      "kube-apiserver-cert-syncer-client-cert-key",
-			Validity:  1 * 4 * time.Hour, // to be 5 days
-			Refresh:   2 * time.Hour,     // to be 1 day
+			Validity:  30 * rotationDay,
+			Refresh:   15 * rotationDay,
```

Comment:
for the refresh I would have expected much shorter time, more like 1 or 2 days.

Reply:
Why would you want to make it that short? I've spoke with @deads2k and for now this value is ok.